jetty-servlets/src/main/java/org/eclipse/jetty/servlets/IncludeExcludeBasedFilter.java

```diff
@@ -120,18 +106,6 @@ protected boolean shouldFilter(HttpServletRequest http_request, HttpServletRespo
             return false;
         }
 
-        String content_type = http_response.getContentType();
```

Comment:
I implemented a fallback to request URI. Let me know if you have any additional feedback. Thanks.